src/core/server/integration_tests/saved_objects/migrations/multiple_kibana_nodes.test.ts

```diff
@@ -161,10 +162,10 @@ describe('migration v2', () => {
 
     if (esServer) {
       await esServer.stop();
+      await delay(10000);
```

Comment:
This 'magic 10sec awaiting' is supposed to be performed after the es server stops. As we usually only boot ES once per test file, we probably copied the `afterAll` block from somewhere. But given that we're rebooting the ES server after each test in this suite, it needed to be in the `after` block instead